packages/core/src/browser/keybinding.spec.ts

```diff
@@ -173,6 +173,109 @@ describe('keybindings', () => {
+        keybindingRegistry.registerKeybinding(otherKeybinding);
+        const numOfKeybindingsForOtherCommand = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND.id).length;
+
+        const keybinding: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: 'F5'
+        };
+        const keybinding2: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: 'F6'
+        };
+
+        keybindingRegistry.registerKeybinding(keybinding);
+        keybindingRegistry.registerKeybinding(keybinding2);
+
+        keybindingRegistry.unregisterKeybinding(TEST_COMMAND2);
+
+        const bindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND2.id);
+        if (bindings) {
+            expect(bindings.length).to.be.equal(0);
+        }
+        const otherBindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND.id);
+        if (otherBindings) {
+            expect(otherBindings.length).to.be.equal(numOfKeybindingsForOtherCommand);
+        }
+    });
+
+    it('should unregister a specific keybinding', () => {
+        const otherKeybinding: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: 'F4'
+        };
+
+        keybindingRegistry.registerKeybinding(otherKeybinding);
+        const keybinding: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: 'F5'
+        };
+
+        keybindingRegistry.registerKeybinding(keybinding);
+
+        keybindingRegistry.unregisterKeybinding(keybinding);
+
+        const bindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND2.id);
+        let otherKeybindingStillThere = false;
+        if (bindings) {
+            bindings.forEach(
+                (value: Keybinding) => {
+                    expect(value.command === keybinding.command && value.keybinding === keybinding.keybinding).to.be.false;
+                    if (value.command === otherKeybinding.command && value.keybinding === otherKeybinding.keybinding) {
+                        otherKeybindingStillThere = true;
+                    }
+                }
+
+            );
+        }
+        expect(otherKeybindingStillThere).to.be.true;
+    }
+    );
+
+    it('should unregister a specific key', () => {
+        const otherKeybinding: Keybinding = {
+            command: TEST_COMMAND.id,
+            keybinding: 'F4'
+        };
+
+        keybindingRegistry.registerKeybinding(otherKeybinding);
+        const numOfOtherKeybindings = keybindingRegistry.getKeybindingsForCommand(TEST_COMMAND.id).length;
+        const testKey = 'F5';
+        const keybinding: Keybinding = {
+            command: TEST_COMMAND2.id,
+            keybinding: testKey
+        };
+
+        const keybinding2: Keybinding = {
+            command: TEST_COMMAND.id,
+            keybinding: testKey
+        };
+
+        keybindingRegistry.registerKeybinding(keybinding);
+        keybindingRegistry.registerKeybinding(keybinding2);
+
```

Comment:
I think an `expect` is missing here. These tests also pass if register/unregister does not work at all.

Reply:
Same as above